docs/manual/developer/02_building_complianceascode.md

```diff
@@ -34,21 +34,19 @@ On *Red Hat Enterprise Linux 7* make sure the packages `cmake`, `openscap-utils`
 `PyYAML`, `python-jinja2`, `python-setuptools` and their dependencies are installed:
 
 ```bash
-yum install cmake make openscap-utils openscap-scanner PyYAML python-jinja2
+yum install $(bindep -b)
 ```
 
 On *Red Hat Enterprise Linux 8* and *Fedora* the package list is the same but python2 packages need to be replaced with python3 ones:
 
 ```bash
-yum install cmake make openscap-utils openscap-scanner python3-pyyaml python3-jinja2 python3-setuptools
+yum install $(bindep -b)
 ```
 
-On *Ubuntu* and *Debian*, make sure the packages `libopenscap8`,
-`libxml2-utils`, `python3-jinja2`, `python3-yaml`, `python3-setuptools`, `xsltproc` and their dependencies are
-installed:
+On *Ubuntu* and *Debian*:
 
 ```bash
-apt-get install cmake make libopenscap8 libxml2-utils ninja-build python3-jinja2 python3-yaml python3-setuptools xsltproc
+apt install $(bindep -b)
```

Comment:
The current `[platform:rpm]` on the dependencies this doesn't work an `apt` based system. Do you plan on adding items for `dpkg` based OSes? If not I can add some.

Reply:
Yep - I can add them.